frontend/src/metabase/collections/components/CreateCollectionForm/CreateCollectionForm.tsx

```diff
@@ -27,7 +27,7 @@ import FormCollectionPicker from "metabase/collections/containers/FormCollection
 import type { Collection } from "metabase-types/api";
 import type { State } from "metabase-types/store";
 
-import FormAuthorityLevelFieldContainer from "./FormAuthorityLevelFieldContainer";
+import FormAuthorityLevelFieldContainer from "../../containers/FormAuthorityLevelFieldContainer";
```

Comment:
nit: worth having `metabase/...` kind of url

Reply:
I think VS Code also kind of automatically changes this to `metabase/*` if it's more than 2 levels from the current component. I have observed that I only get auto import with relative import at a maximum of 2 levels up from the current file.

We should discuss this. IMO, 2 levels is still fine.